Packages/vcs/Lib/fillarea.py

```diff
@@ -37,49 +37,60 @@ def process_src(nm, code):
     except:
         f = vcs.elements["fillarea"][nm]
     atts = {}
-    for a in ["faci", "fasi", "fais", "faoi", "vp", "wc", "x", "y"]:
-        i = code.find(a + "(")
-        v = genutil.get_parenthesis_content(code[i:])
-        # print nm,a,v
-        if v != "":
-            vals = []
-            for V in v.split(","):
-                try:
-                    vals.append(int(V))
-                except:
-                    vals.append(float(V))
-            atts[a] = vals
-        f.style = atts.get("fais", f.style)
+    if code.find("(") > -1:  # ok with have the keywords speeled out
```

Comment:
you don't know about speeling keywords? You should looking into it it's most useful :wink: